src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Data/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -66,7 +66,46 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("DeviceCodes");
                 });
 
-            modelBuilder.Entity("IdentityServer4.EntityFramework.Entities.PersistedGrant", b =>
+            modelBuilder.Entity("Duende.IdentityServer.EntityFramework.Entities.Key", b =>
+                {
+                    b.Property<string>("Id")
+                        .HasMaxLength(450)
+                        .HasColumnType("TEXT");
```

Comment:
@ajcvickers can you double check that these migrations look reasonable to you, I ask because the version that IdentityServer is using here https://github.com/DuendeSoftware/IdentityServer/blob/main/migrations/IdentityServerDb/Migrations/PersistedGrantDb/PersistedGrantDbContextModelSnapshot.cs#L25 is using nvarchar(200), just want to make sure we are using the correct pattern here.

Reply:
Also tagging @bricelam as @ajcvickers might not be able to reply for a bit.

Reply:
Are the tests using SQLite? If so, this is correct--type names are largely irrelevant in SQLite.

Reply:
Great, thanks @bricelam